appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1201,9 +1201,15 @@ definitions:
           - $ref: '#/definitions/SelfLink'
           - $ref: '#/definitions/EditLink'
           - $ref: '#/definitions/ActivityCategoryLink'
+      creationTime:
+        type: string
+        format: dateTime
+        description: The creation time of this goal
+        example: 2016-04-26T20:08:55.365+02:00
```

Comment:
Other examples use this time stamp sample format: ``2016-02-31T21:28:02.290+0000`` (without the colon in the middle). Pleas use that here too.